### PR TITLE
added lesson 10: organizing files

### PR DESCRIPTION
**Pull request by 3tom**
Dear @tomekDiakowTieto ,

Please take a look.

Thank you,
Tomasz

### CODE REVIEW

File: students/medrek_tomasz/lesson_10_organizing_files/selective_copy.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        print_usage()
+        exit()
+
+    file_extension = sys.argv[1]
+    source = os.path.abspath(sys.argv[2])
+    destination = os.path.abspath(sys.argv[3])
+
+    try:
+        os.makedirs(destination)
+    except PermissionError as e:
```
[the remark this answers is not in the document]
Comment (3tom) [reply students/medrek_tomasz/lesson_10_organizing_files/selective_copy.py]: https://github.com/jedzej/tietopythontraining-basic/issues/606

File: students/medrek_tomasz/lesson_10_organizing_files/selective_copy.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            raise
+
+    copied_files = 0
+    for directory, _, files in os.walk(source):
+        for file in files:
+            if file.endswith(file_extension):
+                try:
+                    print(shutil.copy(os.path.join(directory, file),
+                                      destination))
+                except PermissionError as perm_err:
```
[the remark this answers is not in the document]
Comment (3tom) [reply students/medrek_tomasz/lesson_10_organizing_files/selective_copy.py]: https://github.com/jedzej/tietopythontraining-basic/issues/606

File: students/medrek_tomasz/lesson_10_organizing_files/deleting_unneded_files.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            if os.path.getsize(file_path) > size_in_bytes:
+                print(file_path)
+                matched_files += 1
+
+    print("Found {} files in {} larger than {}{}".format(
+        matched_files, dir_path, size, unit))
+
+
+if __name__ == "__main__":
+    main()
```
Comment (tomekDiakowTieto) [inline students/medrek_tomasz/lesson_10_organizing_files/deleting_unneded_files.py]: Perfect :)

File: students/medrek_tomasz/lesson_10_organizing_files/filling_in_the_gaps.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    file_prefix = sys.argv[1]
+    dir_path = os.path.abspath(sys.argv[2])
+
+    if not os.path.isdir(dir_path):
+        print("There is not such directory")
+        exit()
+
+    split_match_files = []
+    for file in os.listdir(dir_path):
+        m = re.search(r"(^{})(\d+(?=.))(.+)".format(file_prefix), file)
```
Comment (tomekDiakowTieto) [inline students/medrek_tomasz/lesson_10_organizing_files/filling_in_the_gaps.py]: Why 'm' ?

**Pull request merged**